src/cljs/rems/administration/create_form.cljs

```diff
@@ -274,10 +269,10 @@
 
 (defn autoscroll []
   (when-let [edit-field (first-partially-visible-edit-field)]
-    (let [index (.getAttribute edit-field "data-field-index")
+    (let [id (last (str/split (.-id edit-field) #"-"))
```

Comment:
should we have a data- attribute for this?

Reply:
I think this will be changed in the PR about the conditional fields so we can check it there.